app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -226,6 +230,7 @@ class PipelineSampleReport extends React.Component {
     this.setState({
       activeThresholds: [Object.assign({}, this.defaultThreshold)],
       excluded_categories: [],
+      exclude_phage: false,
```

Comment:
this is fine for now but I think since you are going to introduce fungi, you probably should put the list into sub_categories and excluding them will become generic. 

Also, right now when you uncheck virus, the phage filter doesn't get unchecked. I think the relationship should be speficied and when people click on exclude virus, the phage filter will be excluded by default. same for fungi/euracryotes in the future.